.github/workflows/prebuild.yaml

```diff
@@ -169,14 +174,19 @@ jobs:
           npm install --ignore-scripts
           . prebuild/macOS/preinstall.sh
           cp prebuild/macOS/binding.gyp binding.gyp
-          node-gyp rebuild -j 2
+          node-gyp rebuild -j 2 --arch=${{ matrix.os.arch }}
```

Comment:
node-gyp defaults to `process.arch`. Is this change needed?

Reply:
Good question - I'm not experienced with `node-gyp` and I think I copied this from someplace else without testing it without that option. I'll try it out without it today.

Reply:
Alright I've pushed a commit that removes that flag, and as far as I can tell, this step of the build process is completing as intended: https://github.com/jamesbvaughan/node-canvas/actions/runs/11115619922/job/30884399250#step:4:178

I should point out though that I'm wading in unfamiliar territory for me when it comes to `node-gyp` and macos binary signing, so I'd really appreciate if someone with more experience with those could help verify that everything looks correct here.

Reply:
Alright I've pushed a commit that removes that flag, and as far as I can tell, this step of the build process is completing as intended: https://github.com/jamesbvaughan/node-canvas/actions/runs/11115619922/job/30884399250#step:4:178

I should point out though that I'm wading in unfamiliar territory for me when it comes to `node-gyp` and macos binary signing, so I'd really appreciate if someone with more experience with those could help verify that everything looks correct here.